src/voting/weighted.ts

```diff
@@ -84,18 +89,23 @@ export default class WeightedVoting {
       )
       .map((arr) => arr.map((pwr) => [pwr]));
 
+    const validScoresTotal = this.getValidVotes().reduce(
+      (a, b: any) => a + b.balance,
+      0
+    );
```

Comment:
The whole problem is with this method `getValidVotes`, now `getScoresByStrategy` and `getScores` return different scores before and after the proposal end while using shutter,

Reply:
I mean with the shutter it is just strings. it may not work with these changes too. if we ignore strings, it changes scores before and after `end`

Reply:
With my changes it won't `getValidVotes ` and `getScoresByStrategy` return zero values only `getScoresTotal` changes once proposal is closed

Reply:
Not really sure what your point is to be honest. Maybe the method names are a bit confusing?

Reply:
This PR is what I was thinking in our call, not sure I understand what you mean @ChaituVR, for Shutter we can use just the function `getScoresTotal`

Reply:
Yep, thanks guys. I now feel completely confused about this 😆 Can ignore my comments